metacat-common-server/src/main/java/com/netflix/metacat/common/server/monitoring/LogConstants.java

```diff
@@ -13,95 +13,109 @@
-    CounterSaveMViewPartitions(AppPrefix + ".countSaveMViewPartitions"),
-    CounterCreateCatalogFailure(AppPrefix + ".countCreateCatalogFailure"),
-    CounterCreateTableFailure(AppPrefix + ".countCreateTableFailure"),
-    CounterCreateDatabaseFailure(AppPrefix + ".countCreateDatabaseFailure"),
-    CounterCreateMViewFailure(AppPrefix + ".countCreateMViewFailure"),
-    CounterDeleteDatabaseFailure(AppPrefix + ".countDeleteDatabaseFailure"),
-    CounterDeleteTablePartitionsFailure(AppPrefix + ".countDeleteTablePartitionsFailure"),
-    CounterDeleteMViewPartitionsFailure(AppPrefix + ".countDeleteMViewPartitionsFailure"),
-    CounterDeleteTableFailure(AppPrefix + ".countDropTableFailure"),
-    CounterDeleteMViewFailure(AppPrefix + ".countDeleteMViewFailure"),
-    CounterGetCatalogFailure(AppPrefix + ".countGetMetadataFailure"),
-    CounterGetCatalogNamesFailure(AppPrefix + ".countGetCatalogNamesFailure"),
-    CounterGetDatabaseFailure(AppPrefix + ".countGetDatabaseFailure"),
-    CounterGetMViewPartitionsFailure(AppPrefix + ".countGetMViewPartitionsFailure"),
-    CounterGetTablePartitionsFailure(AppPrefix + ".countGetTablePartitionsFailure"),
-    CounterGetTableFailure(AppPrefix + ".countGetTableFailure"),
-    CounterGetMViewFailure(AppPrefix + ".countGetMViewFailure"),
-    CounterGetCatalogMViewsFailure(AppPrefix + ".countGetCatalogMViewsFailure"),
-    CounterGetTableMViewsFailure(AppPrefix + ".countGetTableMViewsFailure"),
-    CounterRenameTableFailure(AppPrefix + ".countRenameTableFailure"),
-    CounterUpdateCatalogFailure(AppPrefix + ".countUpdateCatalogFailure"),
-    CounterUpdateTableFailure(AppPrefix + ".countUpdateTableFailure"),
-    CounterSaveTablePartitionsFailure(AppPrefix + ".countSaveTablePartitionsFailure"),
-    CounterSaveMViewPartitionsFailure(AppPrefix + ".countSaveMViewPartitionsFailure"),
+    CounterElasticSearchDatabaseCreate(AppPrefix + ".elasticsearch.count.databaseCreate"),
+    CounterElasticSearchDatabaseDelete(AppPrefix + ".elasticsearch.count.databaseDelete"),
+    CounterElasticSearchTableCreate(AppPrefix + ".elasticsearch.count.tableCreate"),
+    CounterElasticSearchTableDelete(AppPrefix + ".elasticsearch.count.tableDelete"),
+    CounterElasticSearchTableSave(AppPrefix + ".elasticsearch.count.tableSave"),
+    CounterElasticSearchTableRename(AppPrefix + ".elasticsearch.count.tableRename"),
+    CounterElasticSearchTableUpdate(AppPrefix + ".elasticsearch.count.tableUpdate"),
+    CounterElasticSearchPartitionSave(AppPrefix + ".elasticsearch.count.partitionSave"),
+    CounterElasticSearchPartitionDelete(AppPrefix + ".elasticsearch.count.partitionDelete"),
+    CounterElasticSearchDelete(AppPrefix + ".elasticsearch.count.esDelete"),
+    CounterElasticSearchBulkDelete(AppPrefix + ".elasticsearch.count.esBulkDelete"),
+    CounterElasticSearchUpdate(AppPrefix + ".elasticsearch.count.esUpdate"),
+    CounterElasticSearchBulkUpdate(AppPrefix + ".elasticsearch.count.esBulkUpdate"),
```

Comment:
This is to separated the operations from metacat e.g. database/table or elasticSearch itself.  The elasticsearch prefix indicates the components, and the 'es' indicates the operation target.